src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/FeatureFlag/FormTheme/feature_flag_form.html.twig

```diff
@@ -0,0 +1,73 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% block feature_flag_row %}
+  <div class="form-group row feature_flag-widget">
+    <div class="form-control-label">
+      {{ form_label(form) }}
+    </div>
+    <div class="col-sm input-container">
+      {{ form_widget(form) }}
+    </div>
+  </div>
+{% endblock %}
+
+{% block feature_flag_label %}
+  <label for="{{ id }}">{{ label }}</label>
+  <div>
+    <ul class="list-inline small form-text">
+      {% for type in types %}
+        <li class="list-inline-item {% if type == used_type %}font-weight-bold{% endif %}">{{ type }}</li>
+      {% endfor %}
+    </ul>
+  </div>
+{% endblock %}
+
+{% block feature_flag_widget %}
+  {% apply spaceless %}
+  {% set rowAttributes  = row_attr|default({})%}
+  <div class="input-group {{ rowAttributes.class|default('') }}" {% for key, rowAttr in rowAttributes %} {% if key != 'class' %}{{ key }}="{{ rowAttr }}"{% endif %}{% endfor %}>
+    <span class="ps-switch" id="{{ form.vars.id }}">
+        {% for choice in choices %}
+          {% set inputId = id ~'_' ~ choice.value %}
+          <input id="{{inputId}}"
+            {{ block('attributes') }}
+            name="{{ full_name }}"
+            value="{{ choice.value }}"
+            {% if choice is selectedchoice(value) %}checked=""{% endif %}
+            {% if disabled %}disabled=""{% endif %}
+            type="radio"
+          >
+          {% if show_choices %}<label for="{{inputId}}">{{ choice.label|trans({}, choice_translation_domain) }}</label>{% endif %}
+        {% endfor %}
+        <span class="slide-button"></span>
+    </span>
+  </div>
+  {% endapply %}
+  {{- block('form_help') -}}
```

Comment:
Unless I missed a modification it seems like all this is the same as the `switch_widget` no? The you should use the parent widget and only add your custom div after

Reply:
```suggestion
  {{- parent() -}}
```